src/main/java/entities/CurrentUser.java

```diff
@@ -10,6 +10,10 @@ public static User getCurrentUser() {
         return currentUser;
     }
 
+    public static int getCurrentUserId() {
+        return currentUser.getId();
```

Comment:
This is supposed to return the static variable currentUser of type User, not its ID directly!

Reply:
You should merge main into your local so that this is how it is on main.